src/module.api/stats.controller.ts

```diff
@@ -58,9 +67,15 @@ export class StatsController {
       dex = dex.plus(requireValue(liq, `tvl.dex.${pair.symbol}`))
     }
 
+    const optionalUsdt = await this.poolPairService.getUSDT_PER_DFI()
+    const usdt = requireValue(optionalUsdt, 'price.usdt')
+    const masternodes = await this.masternodeStatsMapper.getLatest()
+    const masternodeTvl = requireValue(masternodes?.stats?.tvl, 'masternodes.stats.tvl')
+
     return {
       dex: dex.toNumber(),
-      total: dex.toNumber()
+      total: dex.toNumber(),
```

Comment:
```suggestion
      total: dex.toNumber(),
```

total = dex + masternode